src/commands/dsyms/utils.ts

```diff
@@ -0,0 +1,127 @@
+import {exec} from 'child_process'
+import {mkdirSync} from 'fs'
+import glob from 'glob'
+import path from 'path'
+import {promisify} from 'util'
+import {buildPath} from '../../helpers/utils'
```

Comment:
```suggestion

import {buildPath} from '../../helpers/utils'

```

Reply:
why is that? `/helpers/**` should be separate from the others?

Reply:
We sort dependencies with stdlibs, 3p libs, then all imports from projects files grouped by their amount of `../`. (as in https://github.com/DataDog/datadog-ci/blob/master/src/commands/junit/upload.ts) though it is not enforced by the linter.